main.go

```diff
@@ -373,22 +315,42 @@ func main() {
 	rootCmd.Flags().MarkHidden("reset-state")           //nolint:errcheck
 	rootCmd.Flags().MarkHidden("pause-on-initial-sync") //nolint:errcheck
 
-	statusCmd.Flags().Int("port", DefaultServerPort, "Port number")
+	// MongoDB client timeout (visible: commonly needed for debugging)
+	rootCmd.PersistentFlags().String("mongodb-operation-timeout", "",
+		"Timeout for MongoDB operations (e.g., 30s, 5m)")
+
+	// Bulk write option (hidden: internal tuning)
+	rootCmd.PersistentFlags().Bool("use-collection-bulk-write", false,
+		"Use collection-level bulk write instead of client bulk write")
+	rootCmd.PersistentFlags().MarkHidden("use-collection-bulk-write") //nolint:errcheck
+
+	// Clone tuning options (hidden: advanced tuning, CLI/HTTP only)
+	rootCmd.PersistentFlags().Int("clone-num-parallel-collections", 0,
```

Comment:
Didn't we agreed that cloning options should be added to `pcsm start` subcommand: https://www.notion.so/PCSM-219-Configuration-refactor-2cd674d091f380ceb2b5e51b9e6ded23?d=2ce674d091f3806088e2001c5e78a85e&source=copy_link#2cd674d091f380e88be1e57b6f1c8b8b